Fix for problem with conversion of WiFi IE data (hexstr_2_bytearray)

kimusan opened the pull request:
This pull request fixes a problem found in the wifi api function slsi_hexstr_2_bytearray. It did not convert the hex string correctly in all cases and this corrupts the IE data.

sunghan-chang commented:
Hello @kimusan .
Thank you for contribute, but we don't need to leave a history for "Corrected text about board availabilty" and reverting it.
Could you please leave a changing on wifi?

kimusan commented:
Hi,
Not sure what you want me to do? I have rebased on top of your latest master and removed the two changes for the readme from my history. Anything else I should do?

sunghan-chang commented:
@kimusan No, enough. Thank you.
